### DISCUSSION TITLE
Why these english letters are very wide?

### DISCUSSION THREAD

**Discussion by buptyyf** [pymupdf/PyMuPDF#2262] [Q&A]
Example pdf:
[英国工人阶级状况 .pdf](https://github.com/pymupdf/PyMuPDF/files/10836711/default.pdf)

<img width="614" alt="image" src="https://user-images.githubusercontent.com/13342539/221490794-a5e7626f-8114-41b3-82f8-e83ab4c9a7d4.png">
Text letters in this page are very wide. Why? And How can I convert these letters to normal size?

```
doc = fitz.open('xxx.pdf')
page_dict = doc[6].get_text('dict')
page_dict['blocks'][2]['lines'][0]['spans'][1]['text']
```

**Comment by JorjMcKie** [comment]
Apologies for the tardy answer. I finally stumbled over this again.
There is nothing you can do here, I am afraid. The English text is simply stored like you see it. Probably it was a scan from some original, which was then badly incorporated.
I tried to copy / paste it with Adobe Acrobat and other PDF viewers and it never got any better.
